.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: SDK Kotlin CI
+
+on:
```

Comment:
can we add  [`workflow_dispatch`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) so that we can trigger this manually through the UI when needed?

Reply:
Added.